packages/babel-parser/src/index.ts

```diff
@@ -29,12 +30,11 @@ export type { Expression, File };
 
 export type ParserOptions = Partial<Options>;
 
-export interface ParseError {
-  code: string;
-  reasonCode: string;
-}
+export type ParseError = ParseErrorGeneric<object>;
 export type ParseResult<Result extends File | Expression = File> = Result & {
+  comments: File["comments"];
   errors: null | ParseError[];
+  tokens: File["tokens"];
```

Comment:
```suggestion
  tokens?: File["tokens"];
```
This should be an optional property since, iirc, it will not be attached to the result if the `tokens` is `false`.